packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -537,6 +561,25 @@ const systemPage = (
     <EntityLayout.Route path="/diagram" title="Diagram">
       <EntitySystemDiagramCard />
     </EntityLayout.Route>
+    <EntityLayout.Route path="/relations" title="Diagram (new)">
```

Comment:
Actually I called it "new" because there is already a "Diagram" tab. Not sure if I even have to include the new diagram here anyway. Or do we want to replace to old one here?

Reply:
It's strictly more useful right?